File: hw/chalan/u2/hw19Inheritance/Farmsystem Python/model/Cow.py
from model.FarmAnimal import FarmAnimal

class Cow(FarmAnimal):
    def __init__(self, is_producing_milk, liters_per_day, id, breed, born_on, gender, is_able_to_reproduce, weight, cage):
        super().__init__(id, breed, born_on, gender, is_able_to_reproduce, weight, cage)
        self.is_producing_milk = is_producing_milk
        self.liters_per_day = liters_per_day

    def milk(self):
        return self.liters_per_day

    def __str__(self):
        return f"Cow{{is_producing_milk={self.is_producing_milk}, liters={self.liters_per_day}, parent={super().__str__()}}}"

File: hw/collaguazo/u2/hw19Inheritance/FarSytemJs/model/Cow.js
import { FarmAnimal } from "./FarmAnimal.js";

export class Cow extends FarmAnimal {
    constructor(id, breed, bornOn, gender, isAbleToReproduce, weight, cage, isProducingMilk, litersPerDay) {
        super(id, breed, bornOn, gender, isAbleToReproduce, weight, cage);
        this.isProducingMilk = isProducingMilk;
        this.litersPerDay = litersPerDay;
    }

    toString() {
        return `Cow {
${super.toString()}  Producing Milk: ${this.isProducingMilk}
  Liters Per Day: ${this.litersPerDay}
}`;
    }
}
